.gitignore

```diff
@@ -4,3 +4,15 @@
 *~
 test/node_modules/*
 test/coverage
+Help/new.html
+Help/content_server.html
+History/new.html
+Index/new.html
+Login/new.html
```

Comment:
There may be pages not built from the template but ends with `.html` in the future?